services/led.md

```diff
@@ -7,12 +7,21 @@ A controller for 1 or more monochrome or RGB LEDs connected in parallel.
 
 ## Registers
 
+    rw color @ 0x80 {
+        red: u8
+        green: u8
+        blue: u8
+    }
+
+Sets or gets the color on the LED. Settings the color cancels any animation running on the light.
```

Comment:
I'm not sure. It seems to have the same effect as a constant animation with that color. I feel this just complicates the service.